### PR TITLE
Kjf/sqd intro

### LINKED ISSUE #2110: [ NEW GUIDE ] SQD intro
This guide should cover:

- overview of SQD package
- pip install instructions (and any specific package installation constraints)

Existing docs for reference:
- SQD readme
- SQD installation page

### PR DESCRIPTION
**Pull request by kaelynj**
Closes #2110

### CODE REVIEW

File: docs/guides/_toc.json [hunk 1]
```diff
@@ -505,7 +505,17 @@
...
           "title": "Qiskit addons",
-          "url": "/guides/addons"
+          "children": [
+            {
+              "title": "Qiskit addons overview",
+              "url": "/guides/addons"
+            },
+            {
+              "title": "Sample-based Quantum Diagonalization",
+              "url": "/guides/sqd-intro"
```
Comment (Eric-Arellano) [inline docs/guides/_toc.json:515]: Should be qiskit-addons-sqd. Then additional pages will be qiskit-addons-sqd-my-topic.

File: docs/guides/post-process-results.mdx [hunk 2]
```diff
@@ -17,3 +17,4 @@ unphysical observables.
 ## Guides for post-processing results
 * [Retrieve and save results](./save-jobs)
 * [Visualize results](./visualize-results)
+* [Sample-based Quantum Diagonalization (SQD)](./sqd-intro)
```
Comment (abbycross) [inline docs/guides/post-process-results.mdx:20]:
```suggestion
* [Sample-based quantum diagonalization (SQD)](./sqd-intro)
```

File: docs/guides/sqd-intro.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Sample-based quantum diagonalization
+description: Overview of the Sample-based quantum diagonalization (SQD) workflow
+---
+
+# Sample-based quantum diagonalization (SQD)
+
+Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as a the Hamiltonian of a quantum system, using quantum and distributed classical computing together.
+
+Classical computing is used to process samples obtained from a quantum processor, and to project and diagonalize a target Hamiltonian in a subspace spanned by them.This allows SQD to be robust to samples corrupted by quantum noise and deal with large Hamiltonians, such as chemical systems with millions of interacting terms, beyond the reach of exact diagonalization methods.
```
Comment (abbycross) [inline docs/guides/sqd-intro.mdx:10]:
```suggestion
Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as the Hamiltonian of a quantum system, using quantum and distributed classical computing together.

Classical computing is used to process samples obtained from a quantum processor, and to project and diagonalize a target Hamiltonian in a subspace spanned by them. This allows SQD to be robust to samples corrupted by quantum noise and manage large Hamiltonians, such as chemical systems with millions of interacting terms, beyond the reach of exact diagonalization methods.
```

File: docs/guides/sqd-intro.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The SQD tool can target Hamiltonians expressed as linear combinations of Pauli operators, or second-quantized fermionic operators. The input samples are obtained by quantum circuits defined by the user, which are believed to be good representations of eigenstates (e.g. the ground state) of a target operator. The convergence rate of SQD as a function of the number of samples improves with the sparseness of the target eigenstate.
```
Comment (abbycross) [inline docs/guides/sqd-intro.mdx:12]: we avoid `e.g`, `i.e.`, and so forth - can we replace with "such as" or "for example" here?

File: docs/guides/sqd-intro.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Installation
+
+
+## SQD Procedure
```
Comment (abbycross) [inline docs/guides/sqd-intro.mdx:17]:
```suggestion
## SQD procedure
```

File: docs/guides/sqd-intro.mdx (hunk 4 above)
Comment (kaelynj) [reply docs/guides/sqd-intro.mdx]: Fixed now!

**Comment by Eric-Arellano** [comment]
Is this now stale?

**Pull request closed without merging**